### ISSUE TITLE
It steals the focus

### ISSUE THREAD

**Issue by paddor** [jasonlong/mater#130]

**Comment by jasonlong** [comment]
Could you elaborate on what you're experiencing vs. expected behavior?

**Comment by paddor** [comment]
As soon as the 25 minutes have elapsed, Mater pops up and steals my focus, meaning I can't keep typing in the application I was typing in. I expect it to not steal my focus after the 25 minutes have elapsed.

**Comment by jasonlong** [comment]
Got it, thanks for the clarification. I'll see if this is something I can control in Electron. :thumbsup:

**Comment by jasonlong** [comment]
I looked into this a little and couldn't find a way to have the window appear at the end of the timer but not gain focus. I did, however, try to make this problem less annoying by allowing the Mater window to be dismissed with the <kbd>esc</kbd> key which will also give focus back to the window that previous had it. This change is in the [1.0.9 release](https://github.com/jasonlong/mater/releases/tag/1.0.9).

I'm going to close this for now but can revisit if a different solution is found in the future.
